modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/channel/EventParameter.java

```diff
@@ -67,6 +74,10 @@ public void setEventKey(final String eventKey) {
         this.eventKey = eventKey;
     }
 
+    public TriggerEvent getTriggerEvent() { return triggerEvent; }
+
+    public void setTriggerEvent(TriggerEvent triggerEvent) { this.triggerEvent = triggerEvent; }
```

Comment:
the content of the method and the closing brace should be both in separate lines